src/openrct2/network/NetworkBase.h

```diff
@@ -31,18 +31,28 @@ class NetworkBase : public OpenRCT2::System
 public: // Common
     bool Init();
     void Close();
-    uint32_t GetServerTick();
+
+    constexpr uint32_t GetServerTick() const noexcept
+    {
+        return _serverState.tick;
+    }
```

Comment:
This can't be constexpr and should be in the cpp file.

Reply:
Reverted constexpr